### PR TITLE
begin fork of PGD 5 docs

### PR DESCRIPTION
**Pull request by PJMODOS**
# Documentation for PGD 5

This is a future release branch - specific instructions must be followed when merging, [see below](#merging-this-branch).
**This branch shall not be merged until PGD 5 is released!**

Please use this branch _only_ for changes specific to version 5 -
changes and corrections that apply to 5 should be made in short-lived
branches that can be merged into develop and released promptly.

## Working on this branch

Other people may be editing and altering this branch. Be sure to pull latest before
beginning work - and be aware that this branch may be rebased on a regular basis to
incorporate changes from the previous version. We recommend using `git pull --rebase` or
setting `pull.rebase` to `true` in git config to avoid mistakes,
e.g. run `git config pull.rebase true` on your local clone.

This branch will live for months, but try to merge and delete your branches within hours or days.

For changes that need review or feedback, create a branch based on this branch and
request feedback on a PR before merging your changes into this branch. See:

Please use descriptive messages and comments for your commits and PRs - this allows
other people working on this PR to see at a glance what is changing.

Good: "updated version to 5 in index.mdx" / "correct misspelled option in CLI reference"
Bad: "updated index.mdx" / "fixed typos"

## Incorporating PGD 4 updates

To bring in applicable changes that have been made to PGD 4, rebase
this branch against develop and force-push. Be sure to review changes for areas
that should be updated for 5. When resolving conflicts, defer to 4
except when changes in this branch are necessary to support 5.

## Merging this branch

When PGD 5 is released, follow this procedure to complete the separation of
documenation for 4 and 5:

1. Rebase one last time (see above)
2. Bring the 4 subdirectory back into the source tree:
   ```
   git checkout develop product_docs/docs/pgd/4/
   git commit -m "restore pgd 4 docs"
   git push
   ```
3. Immediately merge this branch into develop.

### CODE REVIEW

**Comment by josh-heyer** [comment]
⚠Rebased! Make sure to run `git pull --rebase` before continuing work on a local copy of this branch.

File: product_docs/docs/pgd/5/routing/index.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+It can be disabled again by setting the same option to `false`.
+
+There are additional group-level options that affect the routing decisions:
+
+- route_writer_max_lag - Maximum lag in bytes of the new write candidate to be
+  selected as write leader, if no candidate passes this, there will be no writer
+  selected automatically.
+- route_reader_max_lag - Maximum lag in bytes for node to be considered viable
+  read-only node (currently reserved for future use).
+- route_writer_wait_flush - Whether to wait for replication queue flush before
```
Comment (smaher-edb) [inline product_docs/docs/pgd/5/routing/index.mdx:40]: I think as per the discussion in [BDR-2752](https://enterprisedb.atlassian.net/browse/BDR-2752) route_writer_wait_flush is not used anymore and can be removed from here.

File: product_docs/docs/pgd/5/routing/index.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+The available option are:
+
+- listen_address - Address the proxy should listen on.
+- listen_port - Port the proxy should listen on.
+- max_client_conn - Maximum number of connections the proxy will accept.
+- max_server_conn - Maximum number of connections the proxy will make to the
+  Postgres node.
+- server_conn_timeout - Connection timeout for server connections.
+- server_conn_keepalive - Keepalive interval for server connections.
+- fallback_groups - Array of groups (in priority order) to route to if the main
```
Comment (smaher-edb) [inline product_docs/docs/pgd/5/routing/index.mdx:73]: I think we should remove this fallback related options from doc...have already removed ref from cli cmds like set-proxy-options and show-proxies

**Comment by josh-heyer** [comment]
⚠Rebased to pick up a couple of files (release notes, overview) that were still in the v4 subdirectory.

**Comment by PJMODOS** [comment]
Did another rebase post 4.3 release to resolve conflicts.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**